pinc/ProofreadingToolbox.inc

```diff
@@ -1,375 +1,88 @@
 <?php
-include_once($relPath.'site_vars.php'); // $wikihiero_dir $utf8_site
 include_once($relPath.'misc.inc'); // attr_safe(), html_safe()
-include_once($relPath.'CharacterSelector.inc');
 include_once($relPath.'faq.inc');
+include_once($relPath.'stages.inc');
+include_once($relPath.'CharacterSelector.inc');
 
-class ProofreadingToolbox
+function draw_tool_button($accesskey, $title, $label, $onclick, $draw = true)
 {
-    public function __construct()
+    if ($draw)
     {
-        $this->initialize_popup_links();
-        $this->initialize_tool_buttons();
-        $this->initialize_tool_links();
-        $this->character_selector = new CharacterSelector();
+        $access_string = $accesskey ? "accesskey='$accesskey'" : '';
```

Comment:
We should wrap `$accesskey` in `attr_safe()` too so the caller can't accidentally pass in invalid HTML.